### ISSUE TITLE
[Bug]: Trace, Console, Timeline disappears after tests succeed / failed

### ISSUE THREAD

**Issue by jasontjahjono** [microsoft/playwright#29826]
### Version

1.40.0

### Steps to reproduce

N/A

### Expected behavior

I expect to see trace, timeline, console after the tests succeed / fails.

### Actual behavior

The whole trace, timeline, console disappears after it fails / succeed.
![Screenshot 2024-03-05 at 2 18 29 PM](https://github.com/microsoft/playwright/assets/67953053/53b6829b-daa6-4e52-9dc7-31141a446e2f)

### Environment

```shell
System:
  OS: macOS 13.6.4
  CPU: (12) arm64 Apple M2 Max
  Memory: 9.54 GB / 64.00 GB
Binaries:
  Node: 20.5.0 - ~/.nvm/versions/node/v20.5.0/bin/node
  npm: 9.8.0 - ~/.nvm/versions/node/v20.5.0/bin/npm
IDEs:
  VSCode: 1.83.0 - /usr/local/bin/code
Languages:
  Bash: 3.2.57 - /bin/bash
npmPackages:
  @playwright/test: ^1.40.0 => 1.41.1
```

**Comment by dgozman** [comment]
@jasontjahjono I believe this should be fixed in the latest 1.42 release. Could you please update and let me know whether that works for you?

**Comment by jasontjahjono** [comment]
it's working now thank you so much!
